test/unit/DssVestTopUp.test.ts

```diff
@@ -22,13 +28,20 @@ describe("DssVestTopUp", function () {
   const slippageToleranceBps = 100;
 
   let topUp: DssVestTopUp;
+
+  let DssVestTopUp: DssVestTopUp__factory;
```

Comment:
```suggestion
  let dssVestTopUpFactory: DssVestTopUpFactory;
```

And drop the eslint ignore statement.

See example: https://github.com/hackbg/chainlink-solutions-lootbox/blob/main/test/Lootbox.ts

Reply:
You may also move the factories to a `before` block outside of this scope like in the example above.